collect_app/src/main/res/values/strings.xml

```diff
@@ -480,4 +480,6 @@
 <string name="encrypted_form">Encrypted form</string>
 <string name="deleted_form">Deleted form</string>
 <string name="exit">Exit</string>
+<string name="instance_scan_completed">Finished scanning instances.</string>
```

Comment:
This should say `Finished scanning`. We have no mention of instances in the UI and I want to be consistent.